frontend/src/v5/ui/components/viewer/cards/tickets/pinDetails/pinDetails.styles.ts

```diff
@@ -38,8 +46,18 @@ export const PinActions = styled.div`
 
 export const PinName = styled(Typography).attrs({
 	variant: 'h5',
-})`
+})<{required:boolean}>`
```

Comment:
In TicketImageComponent, remove the need for the asterisk because I replaced `styled(Typography)` with
```ts
export const PropertyName = styled(FormLabel)`
	${({ theme }) => theme.typography.h5}
	color: ${({ theme }) => theme.palette.secondary.main};
```

(and changed the asterisk color in the theme)
```ts
		MuiFormLabel: {
			styleOverrides: {
				asterisk: {
					color: COLOR.ERROR_MAIN,
				},
			},
		},
```